src/AppBundle/Security/BankIdService.php

```diff
@@ -46,7 +46,8 @@ public function getAccessToken($code)
 
         $accessToken = $client->get($url)->send()->getBody(true);
 
-        $this->getBankIdUser($accessToken);
```

Comment:
You need to add doc block for methods. 
Why method getAccessToken return BankIdUser?